pallets/ceres-launchpad/src/lib.rs

```diff
@@ -119,6 +118,17 @@ pub mod pallet {
 
         /// Weight information for extrinsics in this pallet.
         type WeightInfo: WeightInfo;
+
+        /// to retrieve asset info
```

Comment:
Just to be consistent with other comments
```suggestion
        /// To retrieve asset info
```

Reply:
The same for `AssetInfoProvider` in other pallets.

Reply:
@wer1st updated all comments